src/main/frontend/src/components/ReviewForm.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+
+const ReviewForm = props => {
+    const { register, handleSubmit, errors } = useForm();
+    const onSubmit = data => console.log(data);
+    console.log(errors);
+    
+    
+    return (
+  <div>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <input type="text" name="Comment" placeholder="Comment" ref={register} />
+        <input type="range" placeholder="Rating " name="Rating" ref={register({max: 5, min: 0})} />
+  
+        <input type="submit" />
+      </form>
+  </div>
```

Comment:
The div here should be indented in to the right of the return here, since it is within it.